Area51/Sources/SettingsViewController.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+import SafariServices
+
+class SettingsViewController: UITableViewController {
+
+    override func viewDidLoad() {
```

Comment:
do not need this function if it only calls super

Reply:
Okay! I think this was also auto-generated. I'll remove it.